common/repositories/player_event_speech_repository.h

```diff
@@ -44,7 +44,17 @@ class PlayerEventSpeechRepository: public BasePlayerEventSpeechRepository {
      */
 
 	// Custom extended repository methods here
+	static int64 GetNextAutoIncrementId(Database& db)
+	{
+		auto results = db.QueryDatabase(
+			fmt::format(
+				"SELECT AUTO_INCREMENT FROM information_schema.tables WHERE TABLE_NAME = '{}';",
```

Comment:
We should assume that the server user can't directly query `information_schema` is there a reason we leaned into this versus max + 1 ?

Reply:
Yes.  I ran into a couple of test cases that failed.  If I empty an etl table, the max + 1 will return 1, when in fact the auto_increment may be 100.  Further, once the retention period is hit, the next id via max + 1 will again be different from the auto_increment.